bwapi/include/BWAPI/Player.h

```diff
@@ -643,6 +643,6 @@ namespace BWAPI
     /// @returns true if the unit type requirements are met, and false otherwise.
     ///
     /// @since 4.1.2
-    bool hasUnitTypeRequirement(UnitType unit, int amount = 1) const;
+    bool hasUnitTypeRequirement(const UnitType unit, const int amount = 1) const;

```

Comment:
@kpp Value types don't need to be marked const. It's verbose. (Note that the Type class and everything that extends it is a value type)